packages/engine-rn/templates/platforms/android/app/src/main/java/rnv_template/MainActivity.java.tpl

```diff
@@ -1,12 +1,21 @@
 package {{APPLICATION_ID}};
 
+import android.os.Bundle;
```

Comment:
do we really need this added? Isn't this something that can be injected by {{PLUGIN_ACTIVITY_IMPORTS}} if needed?